Xamarin.Essentials/Launcher/Launcher.shared.cs

```diff
@@ -5,12 +5,28 @@ namespace Xamarin.Essentials
 {
     public static partial class Launcher
     {
-        public static Task<bool> CanOpenAsync(string uri) => PlatformCanOpenAsync(new Uri(uri));
+        public static Task<bool> CanOpenAsync(string uri)
+        {
+            if (string.IsNullOrWhiteSpace(uri))
+            {
+                throw new ArgumentNullException(nameof(uri), $"Uri cannot be empty.");
+            }
+            return PlatformCanOpenAsync(new Uri(uri));
+        }
 
-        public static Task<bool> CanOpenAsync(Uri uri) => PlatformCanOpenAsync(uri);
+        public static Task<bool> CanOpenAsync(Uri uri)
+            => uri != null ? PlatformCanOpenAsync(uri) : throw new ArgumentNullException(nameof(uri));
```

Comment:
Let's make this a non-expression bodied method and also fill in the full argument null exception for devs like the others.

Reply:
Done